tonic-build/src/prost.rs

```diff
@@ -203,6 +205,13 @@ impl Builder {
         self
     }
 
+    /// Generate a file containing the encoded `prost_types::FileDescriptorSet` for protocol buffers
+    /// modules. This is required for implementing gRPC Server Reflection.
+    pub fn file_descriptor_set_path(mut self, path: impl AsRef<Path>) -> Self {
```

Comment:
I wonder if we should just make this an option to add the descriptor to the generated proto file as a constant?

Reply:
@LucioFranco Perhaps I'm misunderstanding this - wouldn't `prost` need to support that too?

Reply:
If we are going through prost to generate this and it puts it into its own file then yes. I may have misunderstood.